h/templates/activity/search.html.jinja2

```diff
@@ -273,25 +273,14 @@
 
     {{ tags_section() }}
 
-    {# if the group has a concept of members display members and join text #}
-    {% if group.members %} 
-      {{ group_users("Members", group.members, group.creator) }}
+    {% if group_users_display_info %}
```

Comment:
Make sure that if this conditional stays here, that the DOM tree won't be broken (at present, a `<section>` is started inside of this conditional that is closed _outside_ of it—this would break layout).